Basic-Video-Chat/README.md

```diff
@@ -4,3 +4,26 @@ Basic Video Chat sample app
 The Basic Video Chat app is a very simple application meant to get a new developer
 started using the OpenTok iOS SDK. For a full description, see the [Basic tutorial at the
 OpenTok developer center](https://tokbox.com/developer/tutorials/ios/basic-video-chat/).
+
+
+Adding the OpenTok library
+==========================
+In this example the OpenTok iOS SDK was not included as a dependency,
+you can do it through Swift Package Manager or Cocoapods.
+
+
+Swift Package Manager
+---------------------
+To add a package dependency to your Xcode project, you should select 
+*File* > *Swift Packages* > *Add Package Dependency* and enter the repository URL:
+`https://github.com/opentok/vonage-client-sdk-video.git`.
+
+
+Cocoapods
```

Comment:
Both swift and objc basic samples have Podfile. So they just need to run `pod install`

Reply:
Updated to just `pod install`.